app/style/TxHistory.less

```diff
@@ -31,97 +31,109 @@
 }
 
 .pending-overview-details {
-  background-color: rgba(12, 30, 62, 0.0296);
+  background-color: #edeff1;
```

Comment:
Use a theme var for this. It's fine to define a new one if the correct one does not yet exist.

Reply:
fixed